WooCommerce/Classes/ViewRelated/Search/Order/OrderSearchStarterViewController.swift

```diff
@@ -106,23 +128,27 @@ extension OrderSearchStarterViewController: UITableViewDelegate {
 
             return message
         }()
-        let ordersViewController = OrdersViewController(
-            title: cellViewModel.name ?? NSLocalizedString("Orders", comment: "Default title for Orders List shown when tapping on the Search filter."),
-            viewModel: OrdersViewModel(statusFilter: cellViewModel.orderStatus),
-            emptyStateConfig: .simple(message: emptyStateMessage, image: .emptySearchResultsImage)
-        )
 
-        navigationController?.pushViewController(ordersViewController, animated: true)
-
-        tableView.deselectSelectedRowWithAnimation(true)
+        let title = cellViewModel.name ?? Localization.defaultOrderListTitle
+        let emptyStateConfig = EmptyStateViewController.Config.simple(message: emptyStateMessage, image: .emptySearchResultsImage)
+
+        if #available(iOS 13, *) {
+            return OrderListViewController(
+                title: title,
+                viewModel: .init(statusFilter: cellViewModel.orderStatus),
+                emptyStateConfig: emptyStateConfig
+            )
+        } else {
+            return OrdersViewController(
+                title: title,
+                viewModel: OrdersViewModel(statusFilter: cellViewModel.orderStatus),
```

Comment:
nit: this could probably be `.init` like the iOS 13+ case?

```suggestion
                viewModel: .init(statusFilter: cellViewModel.orderStatus),
```

Reply:
You got me! Fixed in b43c3dcbb2ba0f20c1ba263de38036bea88ece54.